src/Microsoft.Data/ArrowStringColumn.cs

```diff
@@ -446,7 +446,7 @@ public override GroupBy GroupBy(int columnIndex, DataFrame parent)
             }
         }
 
-        public override BaseColumn FillNulls(object value, bool inPlace = false) => throw new NotSupportedException();
+        public override DataFrameColumn FillNulls(object value, bool inPlace = false) => throw new NotSupportedException();
```

Comment:
(not related to this PR) But ArrowStringColumn should be able to `FillNulls` when `inPlace` is `false`, right? Maybe we should log an issue for that.

Reply:
Filed an issue for this